<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstUsePage</class>
 <widget class="QMainWindow" name="FirstUsePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>First Use Page</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Add Admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Last_Name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_First_Name">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Email">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Gender">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_First_Name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Email">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_Date_Of_Birth">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::Section::YearSection</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Date_Of_Birth">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date Of Birth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Gender">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Last_Name">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password_Confirmation">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Password_Confirmation">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password Confirmation:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Username">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Username">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="error_Email">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error_Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_Username">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error_Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_First_Name">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error_First_Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_Last_Name">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error_Last_Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_Password">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error_Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_Password_Confirmation">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error_Password_Confirmation</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_Gender">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error_Gender</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLanguage">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>עברית</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Register_New_Physiotherapist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>437</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>First Use Page</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ResetButton</tabstop>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>lineEdit_Username</tabstop>
  <tabstop>comboBox_Gender</tabstop>
  <tabstop>dateEdit_Date_Of_Birth</tabstop>
  <tabstop>lineEdit_First_Name</tabstop>
  <tabstop>lineEdit_Last_Name</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>lineEdit_Password_Confirmation</tabstop>
  <tabstop>AddButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
